lapce-core/src/syntax/mod.rs

```diff
@@ -604,11 +604,11 @@ impl Syntax {
                 match highlight {
                     HighlightEvent::Source { start, end } => {
                         if let Some(hl) = current_hl {
-                            if let Some(hl) = SCOPES.get(hl.0) {
+                            if let Some(&hl) = SCOPES.get(hl.0) {
                                 highlights.add_span(
                                     Interval::new(start, end),
                                     Style {
-                                        fg_color: Some(hl.to_string()),
+                                        fg_color: Some(hl.to_owned()),
```

Comment:
I would prefer if we leave all `to_string()`, as all main contributors write code that way

Reply:
Okay, I'll revert to the old implementation.

Actually, at first I wanted to use `to_owned` and `clone` everywhere, but I realized that this would limit flexibility. Apparently in this place I replaced the functions by mistake.

But I still think the call to `to_string` on `&'static str` should be removed.

Reply:
Done

Reply:
I meant all `to_owned()` -> `to_string()`

Reply:
Done :-)?